src/middlewared/middlewared/etc_files/ttys_config.py

```diff
@@ -1,19 +1,13 @@
-import os
-import signal
-import subprocess
+from middlewared.utils import run
```

Comment:
This does not generate any config files anymore, can we drop `ttys` pseudo-service and move it to a middleware plugin?